### DISCUSSION TITLE
Bestbuy Canada giving false positives on PS5 for the last few days

### DISCUSSION THREAD

**Discussion by vansens** [jef/streetmerchant#2615] [Help]
![success-1623344819508](https://user-images.githubusercontent.com/53381210/121568024-135c5500-c9d4-11eb-8dbd-de361f0895bd.png)

Hello
for the last few days, I have been getting emails for a PS5 on BB Canada, but the screenshot is cut off and the PS5 is not on sale.
Are these my timeout settings?

**Comment by Mr-Tech-13** [comment]
I think bestbuy changed some code, and this caused massive issues
